docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,288 @@
+
+Also for most DHIS2 cypress setups, using the `enableAutoLogin` setup helps managing authentication conveniently. In turn, the `enableAutoLogin` setup utility requires the `chromeAllowXSiteCookies` when using Chrome. So in most cases it makes sense to always use this plugin alongside the `networkShim`.
+
+Below is an example of a plugins file with `options` supplied to the `networkShim` and used in conjunction with `chromeAllowXSiteCookies`:
+
+```js
+const {
+    networkShim,
+    chromeAllowXSiteCookies,
+} = require('@dhis2/cypress-plugins')
+
+const options = {
+    hosts: ['https://main-api.com', 'https://secondary-api.com'],
+    staticResources: [
+        '/endpoint-which-always-return-the-same-data',
+        '/another-endpoint-which-always-return-the-same-data',
+    ],
+}
+
+module.exports = (on, config) => {
+    networkShim(on, options)
+    chromeAllowXSiteCookies(on, config)
+}
+```
+
+#### Support file
+
+The `enableNetworkShim` setup should be imported and executed in the [cypress support file](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Support-file). This function accepts no arguments so should simply be called like this `enableNetworkShim()`. Typically, in a DHIS2 platform app, the `enableAutoLogin` setup is also required so individual tests don't require login steps, which means the file should look like this:
+
+```js
+import { enableAutoLogin, enableNetworkShim } from '@dhis2/cypress-commands'
+
+enableAutoLogin()
+enableNetworkShim()
+```
+
+### Package scripts and dependencies
+
+##### `./package.json`
+
+The following `devDependencies` are required:
+
+```json
+"devDependencies": {
+    "@dhis2/cypress-commands": "^#.#.#",
+    "@dhis2/cypress-plugins": "^#.#.#",
+},
+```
+
+And it is recommended to add some convenience commands to the `scripts` section, for example:
+
+```json
+"scripts": {
+    "cy:start": "BROWSER=none yarn start",
+    "cy:open": "wait-on 'http-get://localhost:3000' && cypress open",
+    "cy:run": "wait-on 'http-get://localhost:3000' && cypress run",
+    "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+    "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
+},
+```
+
+_Note that `cy:stub` here is meant to perform a local stub-run, the stub run on CI should be configured differently as explained in the [GitHub workflow](#github-workflow) section._
+
+Some notes:
+
+1. These example scripts assume both `wait-on` and `concurrently` are installed available. If this is not the case, run:
+    ```bash
+    yarn add -D wait-on concurrently
+    ```
+1. As an alternative to using `wait-on` and `concurrently` the package `start-server-and-test` could be used, which is recommended by cypress. An example can be found [here](https://github.com/dhis2/scheduler-app/blob/master/package.json#L14-L16).
```

Comment:
Maybe we should link to a specific point in time, so it's stable: https://github.com/dhis2/scheduler-app/blob/8af8ae7b8b319384f14b510b7017f2beea730f5c/package.json#L14-L16. Also, feel free to omit this if you think it's confusing and everyone should use wait-on and concurrently.

Reply:
Good suggestion, reg. the link, better to link to a snapshot.

Reg. `start-server-and-test`: I definitely don't think I'd want to omit that given the fact it's simpler and recommended by cypress. Instead we should probably switch the example apps to using that and subsequently update the docs to *only* mention that. Since we'd have to also update the example apps, it would bloat the scope of this PR a bit though and that's why I didn't want to do it now.

I've created follow-up issue [CLI-57](https://jira.dhis2.org/browse/CLI-57) in Jira.